ilastik/utility/exportFile.py

```diff
@@ -249,35 +249,38 @@ def create_slicing(axistags, dimensions, margin, feature_table):
         yields also the actual object id
     """
     assert margin >= 0, "Margin muss be greater than or equal to 0"
-    time = feature_table[Default.TimeColumnName].astype(np.int32)
-    minx = feature_table["Bounding Box Minimum_0"].astype(np.int32)
-    maxx = feature_table["Bounding Box Maximum_0"].astype(np.int32)
-    miny = feature_table["Bounding Box Minimum_1"].astype(np.int32)
-    maxy = feature_table["Bounding Box Maximum_1"].astype(np.int32)
-    table_shape = feature_table.shape[0]
-    try:
-        minz = feature_table["Bounding Box Minimum_2"].astype(np.int32)
-        maxz = feature_table["Bounding Box Maximum_2"].astype(np.int32)
-    except ValueError:
-        minz = [0] * table_shape
-        maxz = [1] * table_shape
-
-    indices = list(map(axistags.index, "txyzc"))
-    excludes = indices.count(-1)
-    oid = 1
-    for i in range(table_shape):
-        if time[i] != time[i - 1]:
-            oid = 1
-        # noinspection PyTypeChecker
-        slicing = [
-            slice(time[i], time[i] + 1),
-            _slice_range(minx[i] - margin, maxx[i] + margin, dimensions[1]),
-            _slice_range(miny[i] - margin, maxy[i] + margin, dimensions[2]),
-            _slice_range(minz[i] - margin, maxz[i] + margin, dimensions[3]),
-            slice(None),
-        ]
-        yield [slicing[x] for x in indices][: 5 - excludes], oid
-        oid += 1
+    if len(feature_table) == 0:
+        yield from ()
+    else:
+        time = feature_table[Default.TimeColumnName].astype(np.int32)
+        minx = feature_table["Bounding Box Minimum_0"].astype(np.int32)
+        maxx = feature_table["Bounding Box Maximum_0"].astype(np.int32)
+        miny = feature_table["Bounding Box Minimum_1"].astype(np.int32)
+        maxy = feature_table["Bounding Box Maximum_1"].astype(np.int32)
+        table_shape = feature_table.shape[0]
+        try:
+            minz = feature_table["Bounding Box Minimum_2"].astype(np.int32)
+            maxz = feature_table["Bounding Box Maximum_2"].astype(np.int32)
+        except ValueError:
+            minz = [0] * table_shape
+            maxz = [1] * table_shape
```

Comment:
For consistency with the other values:

```suggestion
            minz = np.zeros(table_shape, np.int32)
            maxz = np.ones(table_shape, np.int32)
```